infrastructure/cli/cmd/config/handle_configuration.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+//AiryConf Contains source data
+type AiryConf struct {
+	Core struct {
+		Apps struct {
+			Sources struct {
+				Twilio struct {
+					AuthToken  string `yaml:"authToken"`
+					AccountSid string `yaml:"accountSid"`
+				}
+				Facebook struct {
+					AppID         string `yaml:"appId"`
+					AppSecret     string `yaml:"appSecret"`
+					WebhookSecret string `yaml:"webhookSecret"`
+				}
+				Google struct {
+					PartnerKey string `yaml:"partnerKey"`
+					SaFile     string `yaml:"saFile"`
+				}
+			}
+		}
+	}
+}
+
+//ParseAiryConf Transforms the airy.conf file content into a struct
+func ParseAiryConf(configFile string) (AiryConf, error) {
```

Comment:
function can be private. Also the name `parseCoreConf` and `CoreConf` maybe? (I'd normally suggest just `parseConf` but I understand it may be too generic in the context)

Reply:
also the file name `conf_parser.go` seems much more accurate

Reply:
:+1: I'd put only `ParseConf` as also enterprise tokens or configuration can be in the same file in the future.